api/routes/ErrorLog.js

```diff
@@ -5,8 +5,7 @@ const { OK, BAD_REQUEST } = require('../constants').STATUS_CODES;
 const { addErrorLog } = require('../util/errorLog');
 
 router.post('/addErrorLog', async (req, res) => {
-  if (!await addErrorLog(req.body.userEmail, req.body.errorTime,
-    req.body.apiEndpoint, req.body.errorDescription)) {
+  if (!await addErrorLog(req)) {
```

Comment:
Instead if req, you could send in req.body so it can be more clean

Reply:
If I do that I get an error. That was one of the first things I tried.

Reply:
You have to figure out why it is giving you an error.